bin/varnishtest/tests/v00019.vtc

```diff
@@ -18,6 +18,11 @@ varnish v1 -errvcl {Unterminated long-string, starting at} {
 	{" }
 }
 
+varnish v1 -errvcl {Unterminated long-string, starting at} {
+	backend b { .host = "127.0.0.1"; }
+	""" ""
```

Comment:
isn't the problem here that even if the string was fine, it'd be in the wrong place. In other words: are you testing the right thing?

Reply:
According to the previous test no as we are testing throwing this specific error.

Reply:
fair enough, I missed that you are checking the error string too